### ISSUE TITLE
Duplicate PostingsFormat entry in the elasticsearch-sql-cli-6.8.23.jar

### ISSUE THREAD

**Issue by avvishen** [elastic/elasticsearch#83296]
### Elasticsearch Version

6.8.23

### Java Version

8u321

### OS Version

Windows 10 server

### Problem Description

Duplicate “META-INF/services/org.apache.lucene.codecs.PostingsFormat” entry in the “elasticsearch-sql-cli-6.8.23.jar”. causing issues with our integration builds when trying to resign artifacts

### Steps to Reproduce

running any signing tool on the elasticsearch zip files

**Comment by luigidellaquila** [comment]
Hi @avvishen 

I'm afraid I'll need more context here. I tried to check the jar file (from https://www.elastic.co/downloads/past-releases/elasticsearch-6-8-23) and everything seems correct. I also tried to sign it with codesign (osx) and I got no errors.

Can you please provide some additional information on how to reproduce?
As an additional note, did you have a chance to verify if the problem is still present in more recent distributions?

Thanks

Luigi

**Comment by avvishen** [comment]
I am using the windows version and this is what I see when i open the jar using an extractor:
![ESDups](https://user-images.githubusercontent.com/86691987/155205372-9cf190fa-4dfb-4d9a-a593-96c78c56a5bd.png)

**Comment by luigidellaquila** [comment]
Hi @avvishen 

thank you for the details and yes, I found it:

```
> unzip -vl elasticsearch-sql-cli-6.8.23.zip  | grep codecs.PostingsFormat
      69  Defl:N       66   4% 01-06-2022 21:32 96d69132  META-INF/services/org.apache.lucene.codecs.PostingsFormat
     852  Defl:N      486  43% 04-21-2020 10:31 cbe45079  META-INF/services/org.apache.lucene.codecs.PostingsFormat
```

I'm investigating, I'll keep you updated

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @avvishen 

I checked latest version and I see the problem is fixed already. Tracking it back, I found it was fixed around v 7.5.

Unfortunately v 6.8 reached end of life (see https://www.elastic.co/support/eol), so I think the best advice I can give you is to upgrade to a more recent version

Thanks

Luigi

**Comment by avvishen** [comment]
thanks for the investigation. Our stack is planning on being on 6.8 for sometime and have no planned upgrades in the coming future as there are lots of breaking changes for us. We dont use this CLI jar file actively, so we will omit this from our package until we upgrade. I am closing this issue.
